Receive sheet v59 design

jvsena42 opened the pull request:
<!-- Closes | Fixes | Resolves #ISSUE_ID -->
<!-- Brief summary of the PR changes, linking to the related resources (issue/design/bug/etc) if applicable. -->
[FIGMA](https://www.figma.com/design/ltqvnKiejWj0JQiqtDf2JJ/Bitkit-Wallet?node-id=37570-213426&t=ujngOLdeVkikfSDj-4)
### Description

This PR applies the v59 design on receive sheet

### Preview

[full_test.webm](https://github.com/user-attachments/assets/aa3efbc9-8a46-4f7a-a05a-ee765de3e3cd)

[small-device.webm](https://github.com/user-attachments/assets/989a31b2-8095-47fb-a28c-681657903576)

[cjit.webm](https://github.com/user-attachments/assets/d5055c0a-94c6-4c01-9760-b1bc8c1ba08f)


### QA Notes

Test:

- [x] Small devices
- [x] CJIT
- [x] No channels -> Geoblocked -> receive lightning -> error toast
- [x] Has channels -> display unified invoice
- [x] Click on details -> change tab -> component should update when the tab changes

<!-- Add testing instructions for the PR reviewer to validate the changes. -->
<!-- List the tests you ran, including regression tests if applicable. -->

jvsena42 commented:
@piotr-iohk 
This test needs to be updated
https://github.com/synonymdev/bitkit-android/actions/runs/19823358673/job/56891351113?pr=485

ovitrif commented on app/src/main/java/to/bitkit/ui/screens/wallets/receive/ReceiveQrScreen.kt:
```diff
@@ -325,39 +369,104 @@ private fun ReceiveQrSlide(
+            .padding(32.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Display("Receive on <accent>spending balance</accent>".withAccent(accentColor = Colors.Purple))
```
Please extract string resources 🙏🏻 .

ovitrif commented:
> I'll take a look on both

For the UI, please compare with bitkit-ios and try to get it at the same level.

ovitrif commented:
PS. Tested and validated all requests from initial review of yesterday, good job 🙏🏻 👏🏻

jvsena42 commented:
Fixed the padding and simplified state handling (claude complicated to much, had to doit manually)

probably fixed this
https://github.com/synonymdev/bitkit-android/pull/485#issuecomment-3611307584


[Screen_recording_20251204_081338.webm](https://github.com/user-attachments/assets/27b00ff9-81ec-4f20-b390-2f80c7329757)

[Screen_recording_20251204_075025.webm](https://github.com/user-attachments/assets/d08b76bd-e41e-4331-9825-855514fe4f12)

ovitrif commented:
> Fixed the padding and simplified state handling (claude complicated to much, had to doit manually)

Was it Opus 4.5 who let us down? I merely want to understand its limits, because it's just hard, it's so powerful…
